### ISSUE TITLE
Problem with the avg() function

### ISSUE THREAD

**Issue by Amelieatge** [debruine/webmorphR#22]
Hi, I've encountered the same issue, have you found a solution or a workaround ?
Thanks in advance !

_Originally posted by @Amelieatge in https://github.com/debruine/webmorphR/issues/21#issuecomment-2033930443_

**Comment by annaabaanana** [comment]
I have encountered the same issue today, any luck finding a solution?

**Comment by Amelieatge** [comment]
No, sorry, I've made no advancement on that matter, good luck ! Let me know if you do!

**Comment by annaabaanana** [comment]
> ```
>           Hi, I've encountered the same issue, have you found a solution or a workaround ?
> ```
> 
> Thanks in advance !
> 
> _Originally posted by @Amelieatge in [#21 (comment)](https://github.com/debruine/webmorphR/issues/21#issuecomment-2033930443)_

Looks like it has been archived on CRAN. Have you found another similar R package that can do similar things?
